site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,433 @@
+  "order": 1,
+  "type": "custom",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "content": {
+    //any json object
+  },
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+A **custom event** conatins a payload that is entirely customizable. It can be used to send any JSON.
+
+## Pushes
+
+Events coming **from a server to a client** are called **pushes**. They are used to keep the application state up-to-date. Pushes are available only in the websocket transport.
+
+As a logged in user, you'll receive server pushes whenever a chat or a thread is started, closed, etc. It applies to both the Agent and Customer Chat API. 
+
+The [incoming event](../agent-chat-api/#incoming-event) push will inform you about the events sent to the chat (on both the agent and customer side).
+
+
+## Properties
+
+> Sample **properties**: 
+
+```js
+{
+    "properties": {
+        "routing": {
+            "pinned": true,
+            "count": 3
+        }
+    }
+}
+```
+Properties are key-value storages. They can be set within a chat, a thread, or an event. 
+
+In our example, `routing` is the namespace, while `pinned` and `count` are properties names.
+
+
+### Configuration
+
+> **Example: using properties to create a basic chat rating.** 
+
+> We need two properties: `rating_score` and `rating_comment`. They should be writable by a Customer, and readable by an Agent in a chat.
+
+> First, we create our properties configuration using the Configuration API.
+
+```
+curl -v https://api.livechatinc.com/configuration/properties/create_properties \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer c5e4f61e1a6c3b1521b541bc5c5a2ac5" \
+    -X POST -d '
+{
+    "rating_score" : {
+        "type" : "int",
+        "locations" : {
+            "chat" : {
+                "access" : {
+                    "agent" : {
+                        "read" : true,
+                        "write" : false
+                    },
+                    "customer" : {
+                        "read" : true,
+                        "write" : true
+                    }
+                }
+            }
+        }
+    },
+    "rating_comment" : {
+        "type" : "string",
+        "locations" : {
+            "chat" : {
+                "access" : {
+                    "agent" : {
+                        "read" : true,
+                        "write" : false
+                    },
+                    "customer" : {
+                        "read" : true,
+                        "write" : true
+                    }
+                }
+            }
+        }
+    }
+}'
+```
+<!-- Description -->
+You can create properties within a licence and configure them using the [Configuration API](../configuration-api/). They are grouped in namespaces, which helps distinguishing which property belongs to a given integration. Your namespace is always named after your `application id`.
```

Comment:
Brakuje informacji/odnośnika tłumaczącego co to jest "application id"

Reply:
W przykladzie jest potem napisane, ze twoim _application_id_ moze byc _client_id_. I tam dodalam gdzie w Consoli mozna znalezc _client_id_. Jak cos wiecej dopisac o samym application_id to potrzebuje konkretnych informacji co :)